docs/develop/json-manifest-overview.md

```diff
@@ -0,0 +1,433 @@
+
+### Conceptual mapping of the preview JSON and current XML manifests
+
+This section describes the preview JSON manifest for readers that are familiar with the current XML manifest. Some points to keep in mind: 
+
+- JSON does not have the XML distinction between attribute and element value. Typically, the JSON that maps to an XML element makes both the element value and each of the attributes a child property. The following example shows some XML markup and its JSON equivalent.
+  
+  ```xml
+  <MyThing color="blue">Some text</MyThing>
+  ```
+
+  ```json
+  "myThing" : {
+      "color": "blue",
+      "text": "Some text"
+  }
+  ```
+- There are many places in the current XML manifest where an element with a plural name has children with the singular version of the same name. For example, the markup to configure a custom menu includes an **Items** element which can have multiple **Item** element children. The JSON equivalent of these plural elements is a property with an array as its value. The members of the array are *anonymous* objects, not properties named "item" or "item1", "item2", etc. The following is an example.
+
+  ```json
+  "items": [
+      {
+          -- markup for a menu item is here --
+      },
+      {
+          -- markup for another menu item is here --
+      }
+  ]
+  ```
+
+#### Top-level structure
+
+The root level of the preview JSON manifest, which roughly corresponds to the **OfficeApp** element in the current XML manifest, is an anonymous object. 
+
+The children of **OfficeApp** are commonly divided into two notional categories. The **VersionOverrides** element is one category. The other consists of all the other children of **OfficeApp**, which are collectively referred to as the base manifest. So too, the preview JSON manifest has a similar division. There is a top-level "extension" property that roughly corresponds in its purposes and child properties to the **VersionOverrides** element. The preview JSON manifest also has over 10 other top-level properties that collectively serve the same purposes as the base manifest of the XML manifest. These other properties can be thought of collectively as the base manifest of the JSON manifest. 
+
+> [!NOTE]
+> When it becomes possible to combine an add-in with other Microsoft 365 extension types in a single web app, then there will be other top-level properties that don't fit into the notion of the base manifest. There will usually be a top-level property for every kind of Microsoft 365 extension type, such as "configurableTabs", "bots" and "connectors". For examples, see the [Teams manifest documentation](/microsoftteams/platform/resources/schema/manifest-schema). This structure makes clear that the "extension" property represents an Office add-in as one type of Microsoft 365 extension.
```

Comment:
"When you combine an add-in with other Microsoft 365 extension types in a single manifest, there will be... "

Suggest updating the "single web app" to "single manifest".  
We've noted that combining and sideloading isn't supported earlier in the doc, so I don't know if we need to call it out again here?